graphics/dcmtk/Portfile

```diff
@@ -66,6 +60,18 @@ configure.args-append   -DDCMTK_WITH_TIFF=OFF \
                         -DDCMTK_WITH_WRAP=ON \
                         -DDCMTK_ENABLE_CXX11=OFF
 
+post-extract {
```

Comment:
I'm not sure that I'm able to reproduce the problem that you work around here.  In particular, Doxygen appears to be scanning the symlinked sources okay.  And the output produced by a scan of the symlinked sources appears to be identical to the output produced by a scan of non-symlinked sources.  Macports is using doxygen version 1.8.14.

Reply:
With this change: (running macports/macports-base@497efcf2) and doxygen 1.8.14 on High Sierra (FWIW):

```sh
$ cd $(port dir dcmtk)
$ sudo port -t destroot +doc
$  find work/destroot/opt/local/share/doc/dcmtk/html/ | wc
    4722    4722  392727
```
without this change:
```sh
$ cd $(port dir dcmtk)
$ sudo port -t destroot +doc
$  find work/destroot/opt/local/share/doc/dcmtk/html/ | wc
      36      36    2112
```

Reply:
Is there a difference in the two logs? Is there an error message from doxygen in the second case?

Reply:
@ryandesign There is a difference.  I was just doing it wrong.  Building dcmtk version 3.6.4 without Eric's post-extract step, generates a Doxyfile, _${workpath}/build/doxygen/htmldocs.tmp_ and invoking `doxygen` on it:
```
cd ${workpath}/build/doxygen && doxygen htmldocs.tmp
```
produces different results, depending on whether _${workpath}/dcmtk-3.6.4_ is a symlink or directory.  

In the case of a symlink, `doxygen` appears to replace the symlink with the actual directory name in its output, whereas the Doxyfile doesn't take this substitution into account.  This may be why no documentation files are produced.  No errors are reported as far as I can see.

If `doxygen` is unable to parse a symlink-rooted tree, that might be a good argument for removing macports/macports-base#55.  Having to Introduce workarounds for `doxygen` would defeat the purpose of that PR.

Or perhaps the Doxyfile could be generated differently?  I'm not familiar with either `cmake` or `doxygen`, so that would require more investigation.

Reply:
Sounds like a bug in doxygen; let's see if we can fix that.

Reply:
In the Doxyfile, replacing the line:
```
INPUT = .../dcmtk-3.6.4 
```
with
```
INPUT = .../dcmtk-3.6.4/.
```
does appear to allow `doxygen` to complete successfully in the case of a symlink.

Reply:
Looks like it has already been reported five months ago: https://github.com/doxygen/doxygen/issues/3518

Reply:
I've moved the work-around to be around Doxygen's unexpected behavior rather than MacPorts's ... six of one...